.travis.yml

```diff
@@ -1,7 +0,0 @@
-language: node_js
```

Comment:
Why are we ditching CI?

Reply:
hmm thinking that this was not being used by pager, but figured it out this project is OS, returning it back

Reply:
exactly, thanks @kuryaki !